package/rig/components/autocomplete/_autocomplete.scss

```diff
@@ -0,0 +1,26 @@
+@import "accessible-autocomplete/src/autocomplete";
+
+// Ensure the autocomplete uses the correct typeface
+.autocomplete__wrapper {
+  font-family: $govuk-font-family;
+}
+
+.autocomplete__input {
+  font-family: inherit;
+}
+
+// Style the autocomplete if there’s an error
+.govuk-form-group--error {
+  .autocomplete__input {
+    border-color: $govuk-error-colour;
+  }
+
+  .autocomplete__input--focused {
+    border-color: $govuk-input-border-colour;
+  }
+}
+
+.autocomplete__dropdown-arrow-down {
+  pointer-events: none;
+  z-index: 0;
+}
```

Comment:
Can you add a comment to this – why is it needed?

Reply:
Done!

Reply:
…although not sure why need to set `z-index: 0`; without it the arrow disappears.

Reply:
Heh – I totally forgot about this:
https://github.com/DFE-Digital/apply-for-teacher-training-prototype/pull/104